packages/next/export/index.ts

```diff
@@ -165,6 +165,22 @@ export default async function exportApp(
     )
   }
 
+  const customRoutesDetected = ['rewrites', 'redirects', 'headers'].filter(
+    (config) => typeof nextConfig[config] === 'function'
+  )
+
+  if (
+    !process.env.NOW_BUILDER &&
```

Comment:
This should use our `isZeitNow()` helper.

Reply:
Updated here https://github.com/vercel/next.js/pull/17630